processors/processExpand.mjs

```diff
@@ -26,6 +26,12 @@ function processExpand(entries, meta) {
    * Place a copy of the entry in the output
    * */
   function expand(e) {
+    //if there are cooridnates, round to one decimal point
+    e.x?e.x=parseFloat(e.x.toFixed(1)):null
+    e.y?e.y=parseFloat(e.y.toFixed(1)):null
+    e.z?e.z=parseFloat(e.z.toFixed(1)):null
+    //generate "key" for entries with x/y and no key
+    if (!e.key && e.x && e.y) e.key = JSON.stringify([e.x,e.y])
```

Comment:
Add braces to if statement

Reply:
Maybe let's set this first and leave it unchanged (without the decimal?)

Reply:
> Maybe let's set this first and leave it unchanged (without the decimal?)

not sure what you mean? you want this to be ints?

Reply:
Yeah, I mean keep the values we were using before, i.e. you can truncate the values to integers for key purposes. I don't actually remember what we use this key for at all

Reply:
i rounded rather than truncating for the extra precision